hack/make/deploy/kubernetes.mk

```diff
@@ -29,26 +37,25 @@ deploy/gke-autopilot: manifests/crd/helm
 			--set manifests=true \
 			--set image="$(IMAGE_URI)" | kubectl apply -f -
 
-
+## Undeploy the operator in a cluster configured in ~/.kube/config where platform and k8s version are autodetected
+undeploy:
+	-helm uninstall dynatrace-operator \
```

Comment:
```suggestion
	helm uninstall dynatrace-operator \
```

Reply:
"-" would swallow the error. We agreed on removing it.